package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<package format="1" xmlns="https://wiki.freecad.org/Package_Metadata">
+  <name>FEM_FrontISTR Workbench</name>
+  <version>0.1.0</version>
+  <date>2021-08-28</date>
+  <description>A FreeCAD addon that enables a parallel nonliner FEM solver FrontISTR.</description>
+  <maintainer email="freecad_support@frontistr.org">FrontISTR-Commons</maintainer>
+  <license file="LICENSE">LGPLv2.1</license>
+  <url type="repository" branch="master">https://github.com/FrontISTR/FEM_FrontISTR</url>
+  <url type="bugtracker">https://github.com/FrontISTR/FEM_FrontISTR/issues</url>
+  <url type="documentation">https://frontistr-commons.gitlab.io/FEM_FrontISTR/en/</url>
+  <icon>Resources/FrontISTR.svg</icon>
+  <depend>FEM</depend>
+  <freecadmin>0.19</freecadmin>
+
+  <content>
+    <workbench>
+      <icon>Resources/FrotISTR.svg</icon>
+      <subdirectory>./</subdirectory>
+      <classname>FemToolsFISTR</classname>
```

Comment:
below line 20, you can add a few <tag>s like:
```
<tag>FEM</tag>
<tag>solver</tag>
```

Reply:
Thanks for your advice! I add them.